lib/zmq-kernel.js

```diff
@@ -47,17 +47,10 @@ export default class ZMQKernel extends Kernel {
       });
 
       this.kernelProcess.stderr.on('data', (data) => {
-        data = data.toString();
-
-        log('ZMQKernel: stderr:', data);
-
-        const regexp = getKernelNotificationsRegExp();
-        if (regexp && regexp.test(data)) {
```

Comment:
We still use this `RegExp` for `stdout`. I think we should make it easier for users and only provide a boolean option which defaults to `false`.

@n-riesco If I recall correctly you added the `RegExp`. What do you think?